repos/system_upgrade/el7toel8/actors/reportsettargetrelease/libraries/reportsettargetrelease.py

```diff
@@ -25,11 +25,10 @@ def _report_set_release():
 def _report_unhandled_release():
     # TODO: set the POST group after it's created.
     target_version = api.current_actor().configuration.version.target
-    commands = [
-        ['subscription-manager', 'release', '--unset'],
-        ['subscription-manager', 'release', '--set', target_version],
-    ]
-    hint = 'Set the new release (or unset it) after the upgrade using subscription-manager.'
+    hint_command = 'subscription-manager release --set {}'.format(target_version)
+    # FIXME: This should use Dialogs and Answers to offer post-upgrade remediation
+    # so that users can choose whether to --set or --unset the release number
+    hint = 'Set the new release (or unset it) after the upgrade using subscription-manager: ' + hint_command
```

Comment:
the used command refers just to the set of the release. What about something like:
```suggestion
    hint_cmd_set = 'subscription-manager release --set {}'.format(target_version)
    hint_cmd_unset = 'subscription-manager release --unset'
    # FIXME: This should use Dialogs and Answers to offer post-upgrade remediation
    # so that users can choose whether to --set or --unset the release number
    hint = 'Set the new release (cmd [1]) or unset it (cmd [2]) after the upgrade using subscription-manager: \n[1] {set_cmd}\n[2] {unset_cmd}'.format(set_cmd=hint_cmd_set, unset_cmd=hint_cmd_unset)
```
Formatting needs a love little bit to meet <120 chars per line.

Reply:
To make it clear, the current messaging is not blocker of this PR. Just pointing this out in case you like that more.